keyboards/hatsukey70/info.json

```
@@ -0,0 +1,109 @@
+{
+    "manufacturer": "interestor",
+    "keyboard_name": "hatsukey70",
+    "maintainer": "interestor",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "processor": "atmega32u4",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0",
+        "transport": {
+            "protocol": "serial"
+        }
```

default setting not required to be explicitly defined 
```suggestion
        "soft_serial_pin": "D0"
```